css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css

```diff
@@ -41,24 +41,40 @@ main {
 
 .box--top-center {
   background-color: #85c1e9;
+  position: relative;
+  top: -80px;
+  left: 210px;
 }
 
 .box--top-right {
   background-color: #5dade2;
+  position: relative;
+  top: -160px;
+  left: 420px;
 }
 
 .box--center {
```

Comment:
http://htmlpreview.github.io/?https://github.com/julie-luciana/challenges/blob/challenges/absolute-position/css-positioning/css-positioning_absolute-no-advanced-requirements/index.html This block is a little closer to the top edge, it's not centred. Please check it.